dist/dist.go

```diff
@@ -0,0 +1,10 @@
+/*
+Distribution the cothority-program in binary form. At least the
+following two distributions should be available:
+
+	* .tar.gz
+	* docker
+
+Others may or may not follow.
+*/
+package dist
```

Comment:
Looks like you mixed up different branches here, or not? I would delete this file (currently doesn't contain anything useful)